packages/form-builder/src/blocks/fields/amount/inspector/index.js

```diff
@@ -0,0 +1,73 @@
+import {PanelBody} from "@wordpress/components";
+import {__} from "@wordpress/i18n";
+import {InspectorControls} from "@wordpress/block-editor";
+import DeleteButton from "./delete-button";
+import AddButton from "./add-button";
+
+const Inspector = (props) => {
```

Comment:
Let's always destruct the props whenever possible.  For this we can have `const Inspector = ({attributes, setAttributes}) => {` instead of referencing the props object everytime.

Reply:
It looks like we can add (or enable) this as an `eslint` rule, see https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md

Reply:
Yeah, unfortunately we're not really using eslint anymore (maybe react scripts does?). It's something i've realized recently that we're missing in our react dev and should start using again.

Reply:
I'm using eslint with create-react-app (but the linked plugin doesn't work with that configuration). For now I'm happy to fix it manually. Resolved in https://github.com/impress-org/givewp-next-gen/pull/49/commits/6070228495fbe8f755fce7bdead42aa88be9adf3.